core/trino-main/src/test/java/io/trino/execution/BaseDataDefinitionTaskTest.java

```diff
@@ -122,6 +123,14 @@ public void setUp()
         queryStateMachine = stateMachine(transactionManager, createTestMetadataManager(), new AllowAllAccessControl(), testSession);
     }
 
+    @AfterMethod
```

Comment:
```java
@AfterClass(alwaysRun = true)

Reply:
This needs to be after method to match the before method above, but I added the always true